lib/beacon_web/components/layouts/runtime.html.heex

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html lang="en" phx-socket={live_socket_path(assigns)}>
   <head>
-    <%= meta_tags(assigns) %>
+    <meta name="csrf-token" content={get_csrf_token()} />
```

Comment:
Is it okay to persist the CSRF token across page navigation? I am genuinely asking because I am not sure. The CSRF token is generated as part of the Plug lifecycle (disconnected mount), so I worry about it breaking if we don't update it during a live navigation.

Reply:
This file is the actual root layout of the host application so it works the same as a regular Phoenix app. It gets this name to make it more explicit and to differentiate from the admin layout.

Reply:
Ah! That makes sense. Thank you.